.gitignore

```diff
@@ -112,3 +112,5 @@ dist
 # Jest
 jest_html_reporters.html
 reports
+
+local.json
```

Comment:
mb local*.json ?
and new line is missing.

Reply:
done.